autosubmit/platforms/wrappers/wrapper_factory.py

```diff
@@ -193,57 +193,6 @@ def header_directives(self, **kwargs):
         return ""
 
 
-class LocalWrapperFactory(WrapperFactory):
```

Comment:
Duplicate (look above this lines in `master`, to see the exact copy of this class re-declared).